src/admin-guide/environments/live-full.yml

```
@@ -0,0 +1,35 @@
+name: cylc-8-live-with-rose
+# Cylc 8, the UI server, Cylc Rose and Metomi Rose
+# n.b. The UI needs to be installed with Yarn: For details see
+# https://github.com/cylc/cylc-ui
+channels:
+  - defaults
+  - conda-forge
+  - kinow
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _openmp_mutex=4.5=1_gnu
+  - ca-certificates=2021.7.5=h06a4308_1
+  - certifi=2021.5.30=py39h06a4308_0
+  - ld_impl_linux-64=2.35.1=h7274673_9
+  - libffi=3.3=he6710b0_2
+  - libgcc-ng=9.3.0=h5101ec6_17
+  - libgomp=9.3.0=h5101ec6_17
+  - libstdcxx-ng=9.3.0=hd4cf53a_17
+  - ncurses=6.2=he6710b0_1
+  - openssl=1.1.1k=h27cfd23_0
+  - pip=21.2.4=py37h06a4308_0
+  - python=3.9.6=h12debd9_1
+  - readline=8.1=h27cfd23_0
+  - setuptools=52.0.0=py39h06a4308_0
+  - sqlite=3.36.0=hc218d9a_0
+  - tk=8.6.10=hbc83047_0
+  - tzdata=2021a=h5d7bf9c_0
+  - wheel=0.37.0=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
```

I think this should reduce to:

```suggestion
  - python=3.9
  - pip
```